File: data/evaluation/geeks_for_geeks_successful_test_scripts/python/FIND_SUM_MODULO_K_FIRST_N_NATURAL_NUMBER_1.py
# Copyright (c) 2019-present, Facebook, Inc.
# All rights reserved.
#
# This source code is licensed under the license found in the
# LICENSE file in the root directory of this source tree.
#
def f_gold ( N , K ) :
    ans = 0 ;
    y = N / K ;
    x = N % K ;
    ans = ( ( K * ( K - 1 ) / 2 ) * y + ( x * ( x + 1 ) ) / 2 ) ;
    return int ( ans ) ;


#TOFILL

if __name__ == '__main__':
    param = [
    (40,90,),
    (46,64,),
    (97,20,),
    (63,1,),
    (92,52,),
    (60,35,),
    (67,40,),
    (61,62,),
    (74,61,),
    (67,41,)
        ]
    n_success = 0
    for i, parameters_set in enumerate(param):
        if f_filled(*parameters_set) == f_gold(*parameters_set):
            n_success+=1
    print("#Results: %i, %i" % (n_success, len(param)))

File: data/evaluation/geeks_for_geeks_successful_test_scripts/cpp/FIND_SUM_MODULO_K_FIRST_N_NATURAL_NUMBER_1.cpp
// Copyright (c) 2019-present, Facebook, Inc.
// All rights reserved.
//
// This source code is licensed under the license found in the
// LICENSE file in the root directory of this source tree.
//

#include <iostream>
#include <cstdlib>
#include <string>
#include <vector>
#include <fstream>
#include <iomanip>
#include <bits/stdc++.h>
using namespace std;
int f_gold ( int N, int K ) {
  int ans = 0;
  int y = N / K;
  int x = N % K;
  ans = ( K * ( K - 1 ) / 2 ) * y + ( x * ( x + 1 ) ) / 2;
  return ans;
}


//TOFILL

int main() {
    int n_success = 0;
    vector<int> param0 {40,46,97,63,92,60,67,61,74,67};
    vector<int> param1 {90,64,20,1,52,35,40,62,61,41};
    for(int i = 0; i < param0.size(); ++i)
    {
        if(f_filled(param0[i],param1[i]) == f_gold(param0[i],param1[i]))
        {
            n_success+=1;
        }
    }
    cout << "#Results:" << " " << n_success << ", " << param0.size();
    return 0;
}